### PR TITLE
fix: add Chaitanya

### PR DESCRIPTION
**Pull request by Chaitanya7666**
I have added my social media data to the public folder.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/881"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
Could you please revert the `package-lock.json` file? There's no need to change that file.

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add Chaitanya7666`

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
### Changes 
 - revert `package-lock.json` chnages.

File: public/data/Chaitanya7666.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
```
Comment (krishguptadev) [inline public/data/Chaitanya7666.json:24]: I suggest a little touch to make it a bit more like a sentence than a heading 😉 

```suggestion
      "description": "Started freelancing!"
```

**Comment by Chaitanya7666** [comment]
Hey folks! I have made the suggested changes. Could you please merge the branches now? Thanks for your suggestions!

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

**Review by nathgoutam93 — APPROVED** [review]

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
Thanks for your contribution @Chaitanya7666

**Pull request merged**